src/services/services.ts

```diff
@@ -1967,6 +1967,76 @@ namespace ts {
         }
     }
 
+    function isObjectLiteralPropertyDeclaration(node: Node): node is ObjectLiteralElement  {

```

Comment:
OK, I got the names backward then.

---
In reply to: [88125144](https://github.com/Microsoft/TypeScript/pull/12107#discussion_r88125144) [](ancestors = 88125144,87306417)